src/views/account-detail/index.js

```diff
@@ -509,6 +601,77 @@ class AccountLatest extends Component {
+                            <Input
+                                className="reveal-pk-passphrase"
+                                style={{ marginTop: "8px", width: "350px" }}
+                                type="password"
+                                name="reveal-pk-passphrase"
+                                id="reveal-pk-passphrase"
+                                placeholder="Passphrase"
+                                minLength="1"
+                            />
+                            <div id="private-key-container" style={{display: "none"}}>
+                                <label id="private-key-label" className="passphrase-label" htmlFor="private">
+                                    PRIVATE KEY
+                                </label>
+                                <Input style={{ 
+                                    backgroundColor: "#f5f5f5", 
+                                    height: "20px",
+                                    width: "350px",
+                                    marginTop: "9px"
+                                    }} type="text" name="private-k" id="private-key-input" defaultValue={""} readonly />
+                                <span style={{marginTop: "18px"}} className="copy-button" onClick={this.copyPrivateKey}> <img src={copy} alt="copy" /></span>
+                            </div>
+                        </div>
+                        <span id="passphrase-invalid" className="error" style={{ display: "none" }}>
+                            {" "}
+                            <img src={altertR} alt="alert" />
+                            Invalid Passphrase
+                        </span>
+                        <Button
+                            style={{
+                                textAlign: "center",
+                                width: "119px", 
+                                display: "block",
+                                padding: "9px 6px",
+                                margin: "24px auto 10px auto" }}
+                            onClick={this.revealPrivateKey.bind(this)}
+                        >
+                            Reveal
+                        </Button>
+                        <button className="close" onClick={this.closeModal.bind(this)}>
+                            <img src={exit} alt="exit icon close"/>
+                        </button>
+                        <div className="alert">
+                                <img src={altertT} alt="alert" />
+                                <div className="cont-alert">
+                                    <div className="title">
+                                        <h3>SAVE YOUR PRIVATE KEY!</h3>
+                                    </div>
+                                    <p>
+                                        You wont be able to reveal it again or restore it. Make a back up, store it save  preferably offline. You’ll need it to acces your account.
```

Comment:
should be

Make a back up and store it safely, preferably offline. You’ll need it to access your account.